arangod/RestServer/Metrics.h

```diff
@@ -459,6 +460,13 @@ template<typename Scale> class Histogram : public Metric {
     } else {
       _c[pos(t)] += n;
     }
+    value_type tmp = _s.load(std::memory_order_relaxed);
+    do {
+    } while (!_s.compare_exchange_weak(tmp,
```

Comment:
I follow the guidelines to use `_weak` if there is a loop anyway. Citing cppreference:

"When a compare-and-exchange is in a loop, the weak version will yield better performance on some platforms.

When a weak compare-and-exchange would require a loop and a strong one would not, the strong one is preferable unless..."

I would need a loop here anyway, since it can happen that the value we read is changed by somebody else.